packages/common/src/core/schemas/internal/filterSchemaToUiSchema.ts

```diff
@@ -0,0 +1,18 @@
+import { getUiSchemaProps } from "./getUiSchemaProps";
+import { IConfigurationSchema, IUiSchema } from "../types";
+import { subsetSchema } from "./subsetSchema";
+
+/**
+ * Subset a json schema to only those properties needed by the uiSchema
+ * @param schema
+ * @param uiSchema
+ * @returns
+ */
+export function filterSchemaToUiSchema(
```

Comment:
The "rule" issue described above also appears to create the same issue here

Reply:
Resolved in lower-level fn.